src/vm/codeman.h

```diff
@@ -140,6 +140,10 @@ typedef struct _hpCodeHdr
     PTR_EE_ILEXCEPTION  phdrJitEHInfo;
     PTR_BYTE            phdrJitGCInfo;
 
+#if defined(FEATURE_GDBJIT)
+    VOID*            pCalledMethods;
```

Comment:
As long as it is tucked behind the ifdef that is off by default it seems fine to me. If it was intended to be on by default in the future then I'd have the same concern as Jan.